cmd/postpolicyform.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/bcicen/jstream"
 	"github.com/minio/minio-go/v7/pkg/set"
+	http2 "github.com/minio/minio/internal/http"
```

Comment:
```suggestion
	xhttp "github.com/minio/minio/internal/http"
```

not `http2` http2 means http2.0